packages/orchestration/test/chain-info.test.js

```diff
@@ -15,6 +15,7 @@ test('chain-info', async t => {
   });
   t.deepEqual(chainNames.keys(), [
     'agoric',
+    'agoriclocal',
```

Comment:
having something that's also a chainid show up here is a bit of a head-tilter. XXX again?